### PR TITLE
[TC-9205] Add simple endpoint

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-9205

Review: https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/tvFBACDFyn1HAqKgktzz/processes/order/buyer/issue

### Scope
This PR
- adds the simple endpoint for simple delivery schedule's
- the simple endpoint for simple delivery history's will be added in [TC-9120](https://tradecloud.atlassian.net/browse/TC-9120)
- `{% api-method ...`'s are replaced by links to the API specs, as they are confusing, incomplete and out of date. The API specs are the single source of truth regarding endpoint specifications

### Submitter pre-flight check

- [ ] Review your documentation
- [ ] Add labels `needs reviewing`
- [ ] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

[TC-9120]: https://tradecloud.atlassian.net/browse/TC-9120?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: order/buyer/cancel.md [hunk 1]
```diff
@@ -12,118 +12,36 @@ Cancel an order line in Tradecloud when it is cancelled at the buyer.
...
-{% page-ref page="update.md" %}
-
 ## Cancelling by resending the order without the cancelled line using the `/order` API
 
-You can cancel an order line by using the `/order` API resource, by setting the `indicators.cancelLineWhenMissing` on order level and updating the order WITHOUT including the cancelled line.
-
-You can cancel a full order \(all the lines\) by setting the `indicators.cancelLineWhenMissing` on order level and updating the order WITHOUT any lines.
-
-This is useful when your ERP system cannot cancel a line, but instead removes a line from the order.
+When your ERP system cannot cancel a line, but instead removes a line from the order, you still still cancel an order line by setting the `indicators.cancelLineWhenMissing` on order level and updating the order WITHOUT including the cancelled line using the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) endpoint:
```
Comment (roy-tc) [inline order/buyer/cancel.md:27]:
```suggestion
When your ERP system cannot cancel a line, but instead removes a line from the order, you can still cancel an order line by setting the `indicators.cancelLineWhenMissing` on order level and updating the order WITHOUT including the cancelled line using the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) endpoint:
```

**Review by roy-tc — COMMENTED** [review]
Looks good, but I don't get why some document features are marked as not available yet..

**Review by Sjorstradecloud — APPROVED** [review]
LGTM

File: order/supplier/send-order-response/attach-document.md [hunk 2]
```diff
@@ -3,200 +3,51 @@ description: How to attach a document to an order or line response
...
-
-* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company permissions.
-* `buyerAccountNumber`: the buyer account number as known in your ERP system
-* `purchaseOrderNumber`: the purchase order number as sent by the buyer
-* `documents`: the documents to be attached to this order.
+You can attach the uploaded document using two methods:
 
 {% hint style="warning" %}
-The total number of documents is limited to 100 documents per order header.
+This feature is not available yet.
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/supplier/send-order-response/attach-document.md:28]: Right, I reviewed code and indeed merging in document is supported but `/order-response/documents` is non-existent.
I will amend this page.

**Comment by marcmatt** [comment]
@roy-tc please review https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/l3BrZblmxBWP7L1VhC9s/processes/order/supplier/send-order-response/attach-document

**Comment by marcmatt** [comment]
@roy-tc Please review again, last version https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/tvFBACDFyn1HAqKgktzz/processes/order/buyer/issue

File: order/buyer/issue/README.md [hunk 3]
```diff
@@ -13,9 +13,15 @@ As buyer you can send either a **new or** [**updated**](../update.md) purchase o
 The new order lines will have order process status `Issued`and logistics status `Open`
 {% endhint %}
 
-## Endpoint
+## Endpoints
 
-Use the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) endpoint to send an order to Tradecloud.
+Use the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) endpoint when your ERP system supports a delivery schedule natively.
+
+Or use the [Send simple order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSimpleOrderByBuyerRoute) for the simple delivery schedule.
```
Comment (roy-tc) [inline order/buyer/issue/README.md:20]:
```suggestion
Or use the [Send simple order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSimpleOrderByBuyerRoute) endpoint for the simple delivery schedule.
```

File: order/buyer/update.md [hunk 4]
```diff
@@ -18,14 +18,14 @@ Most supplier ERP integrations do not have the capability to automatically proce
...
+Using the [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) or [Send simple order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSimpleOrderByBuyerRoute) endpoint is the same as when [sending a new order](issue/) with additional JSON objects as mentioned below. 
+
+Tradecloud will update the order based on the `purchaseOrderNumber` and will add or update lines, delivery schedules and delivery histories where needed. Lines will be matched based on `lines.position`, `deliverySchedule.position` and `deliveryHistory.position`.
 
 {% hint style="info" %}
 The order update should preferable only contain **order lines** that are **new or changed**. But you can also send all lines anyway.
-
-Sending all lines may result in processing delays and unnecessary network, server and storage resource usage.
```
Comment (roy-tc) [inline order/buyer/update.md:28]: I think it's still good to leave this in..

File: order/supplier/send-order-response/attach-document.md [hunk 5]
```diff
@@ -20,34 +16,71 @@ You can attach documents using three methods:
...
+* `description` : optional document description or long name
+* `type`: optional document business type
+* `url`: the document location on the company's content server
+
 ## Method 3. Only provide document meta data
 
 Only provide document meta data like code, revision, name, description and type.
 
-Choose an appropriate API to attach the document:
+You can provide the meta data using the `documents` field on either `order` or `lines` level in the order body of the [Send order response](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/supplier-endpoints/sendOrderResponseBySupplierRoute) endpoint.
```
Comment (roy-tc) [inline order/supplier/send-order-response/attach-document.md:73]: This too will NOT create an activity or workflow task for the buyer.

Perhaps it's good to mention that if you want this activity / task to happen, the supplier should attach a document through the portal?

**Review by roy-tc — COMMENTED** [review]
Looks good, couple of small suggestions

File: order/buyer/update.md (hunk 4 above)
Comment (marcmatt) [reply order/buyer/update.md:28]: As discussed it may be removed

**Pull request merged**